genestack_client/files_util.py

```diff
@@ -637,17 +637,17 @@ def get_differential_expression_stats(self, accessions_to_queries):
 
                 - featureType
                 - featureId
-                - subregions (list)
+                - subregions (list)  (``[]`` if unavailable)
                 - featureName
                 - location (dict):
 
-                    - to
-                    - from (0-based)
-                    - contigName
-                    - normalizedContigName
+                    - to (``-1`` if unavailable)
```

Comment:
yes in the case of microarrays, we don't have information about where the genes map onto the genome, so `from` and `to` are not there